lib/tasks/stack_check.rake

```diff
@@ -7,9 +7,8 @@ namespace :stacks do
     next unless db.table_exists?(:buildpack_lifecycle_data)
 
     RakeConfig.config.load_db_encryption_key
```

Comment:
**suggestion/question** Since we're not loading the models, I don't think we need need to load the encryption keys and nothing in the stacks table is encrypted either.

Reply:
@xandroc / @sethboyles can you confirm if my understanding is right?

Reply:
Yep, we shouldn't need this (or `buildpack_lifecycle_data`) anymore